docker/tinode/entrypoint.sh

```diff
@@ -140,10 +120,10 @@ args=("--config=${CONFIG}" "--static_data=$STATIC_DIR")
 
 # Maybe set node name in the cluster.
 if [ ! -z "$CLUSTER_SELF" ] ; then
-  args+=("--cluster_self=$CLUSTER_SELF")
+	args+=("--cluster_self=$CLUSTER_SELF")
```

Comment:
The check is redundant. Empty parameter `--cluster_self=` is equivalent to no parameter.